demo/api.html

```diff
@@ -1,11 +1,212 @@
+        <p>
+          To disable a rule, you need to remove it from the router or the other rule where it is, if you remove a rule
+          that contains children rules, all the children rules will also be disabled.
+        </p>
+          <pre class="prettyprint lang-js"><code>// Remove from pushStateTree to disable
+pushStateTree.removeChild(myRuleElement);</code></pre>
+
+        <h2 id="rule-properties">
+          Properties
+          <a href="rule-properties" class="anchorjs-link"><span class="anchorjs-icon"></span></a>
+        </h2>
+        <p>
+          You can define the properties passing a literal object with options to the method <code>createRule</code> from
+          the router instance, or you can set those properties in the rule instance after it's created.
+        </p>
+        <pre class="prettyprint lang-js"><code>var pushStateTree = new PushStateTree();
+
+// Create a empty Rule element
+var myRuleELement = pushStateTree.createRule();
+
+// Set properties:
+myRuleELement.id = 'myFirstRule';
+myRuleELement.rule: /.+/;
+
+// Append to pushStateTree to enable
+pushStateTree.appendChild(myRuleElement);</code></pre>
+        <h3 id="rule-property-rule">
+          rule = {Regular Expression}
+          <a href="rule-property-rule" class="anchorjs-link"><span class="anchorjs-icon"></span></a>
+        </h3>
+        <p>
+          This property must contain a Regular Expression that will evaluate the rule element. The context used to match
+          will depend on the parent element.
+        </p>
+        <p>
+          If the parent element is a <code>Router</code> the content will be the full URL not including the
+          <code>basePath</code>.
+        </p>
+        <p>
+          If the parent element is a <code>Rule</code> the content will be the partial URL used by the parent rule, or
+          if the property <code>parentGroup</code> is set, it will use the maching group from the parent rule.
+        </p>
+        <p>
+          In the next example, each string representing the match will be value of a group starting from 0.
+        </p>
+        <pre class="prettyprint lang-js"><code>// Testing address: http://localhost/abc:def/
+myRuleElement1.rule = /.+/;          // "abc:def/"
+myRuleElement2.rule = /abc:(.*)\//;  // "abc:def/", "def"
+myRuleElement3.rule = /(.+):(.+)\//; // "abc:def/", "abc", "def"
+myRuleElement4.rule = /(test)/;      // not match
+
+// Testing address: http://localhost/abc:/
+myRuleElement1.rule = /.+/;          // "abc:/"
+myRuleElement2.rule = /abc:(.*)\//;  // "abc:/", ""
+myRuleElement3.rule = /(.+):(.+)\//; // not match
+myRuleElement4.rule = /(test)/;      // not match
+
+// Testing address: http://localhost/test/
+myRuleElement1.rule = /.+/;          // "test/"
+myRuleElement2.rule = /abc:(.*)\//;  // not match
+myRuleElement3.rule = /(.+):(.+)\//; // not match
+myRuleElement4.rule = /(test)\//;    // "test/", "test"</code></pre>
```

Comment:
this regexps differs from the previous examples. you should keep the same regexps, no?